libp2p/protocols/pubsub/gossipsub.nim

```diff
@@ -244,7 +794,12 @@ proc heartbeat(g: GossipSub) {.async.} =
     except CancelledError as exc:
       raise exc
     except CatchableError as exc:
-      warn "exception ocurred in gossipsub heartbeat", exc = exc.msg
+      warn "exception ocurred in gossipsub heartbeat", exc = exc.msg, trace = exc.getStackTrace()
+      assert(false, "exception ocurred in gossipsub heartbeat")
```

Comment:
remove before merge

Reply:
yep I think there is another one at least as well.